src/pt/wolftoon/src/eu/kanade/tachiyomi/extension/pt/wolftoon/Wolftoon.kt

```diff
@@ -0,0 +1,248 @@
+                                else -> mangas
+                            }
+                        }
+                        else -> {}
+                    }
+                }
+
+            MangasPage(mangas.map { it.toSManga() }, hasNextPage = false)
+        }
+    }
+
+    override fun searchMangaParse(response: Response) = throw UnsupportedOperationException()
+
+    // ================================ Details =======================================
+
+    override fun getMangaUrl(manga: SManga): String = "$baseUrl${manga.url}"
+
+    override fun mangaDetailsRequest(manga: SManga): Request {
+        val titleId = getMangaUrl(manga).toHttpUrl().fragment
+        val url = "$supabaseUrl/rest/v1/titles".toHttpUrl().newBuilder()
+            .setQueryParameter("select", "*")
+            .setQueryParameter("id", "eq.$titleId")
+            .build()
+        return GET(url, apiHeaders)
+    }
+
+    override fun mangaDetailsParse(response: Response): SManga =
+        response.parseAs<List<MangaDto>>().first().toSManga()
+
+    // ================================ Chapters =======================================
+
+    override fun chapterListRequest(manga: SManga): Request {
+        val titleId = getMangaUrl(manga).toHttpUrl().fragment
+        val url = "$supabaseUrl/rest/v1/chapters".toHttpUrl().newBuilder()
+            .addQueryParameter("select", "id,title_id,chapter_number,created_at")
+            .addQueryParameter("title_id", "eq.$titleId")
+            .addQueryParameter("order", "chapter_number.desc")
+            .fragment(titleId)
+            .build()
+
+        return GET(url, apiHeaders)
+    }
+
+    override fun chapterListParse(response: Response): List<SChapter> =
+        response.parseAs<List<ChapterDto>>().map { it.toSChapter() }
+
+    // ================================ Pages =======================================
+
+    override fun fetchPageList(chapter: SChapter): Observable<List<Page>> {
+        return Observable.fromCallable {
+            val chapterUrl = getChapterUrl(chapter).toHttpUrl()
+            val chapterId = chapterUrl.fragment
+            val titleId = chapterUrl.pathSegments[1]
+            val url = "$supabaseUrl/rest/v1/chapters".toHttpUrl().newBuilder()
+                .addQueryParameter("select", "id,title_id,images")
+                .addQueryParameter("title_id", "eq.$titleId")
+                .addQueryParameter("order", "chapter_number.desc")
+                .fragment(chapterId)
+                .build()
+            pageListParse(client.newCall(GET(url, apiHeaders)).execute())
+        }
+    }
+
+    override fun pageListParse(response: Response): List<Page> {
+        val chapterId = response.request.url.fragment
+        return response.parseAs<List<PageDto>>()
+            .firstOrNull { it.id == chapterId }
+            ?.toPageList()
+            ?: emptyList()
+    }
+
+    override fun imageUrlParse(response: Response): String = ""
```

Comment:
```suggestion
    override fun imageUrlParse(response: Response) = throw UnsupportedOperationException()
```

per contributing guidelines.